### ISSUE TITLE
`cargo build` always compiles 32-bit binaries on windows

### ISSUE THREAD

**Issue by vasily-kirichenko** [rust-lang-nursery/rustup.rs#503]
```
> rustup show
installed toolchains
--------------------

stable-i686-pc-windows-gnu (default)
beta-i686-pc-windows-gnu
nightly-i686-pc-windows-gnu

active toolchain
----------------

stable-i686-pc-windows-gnu (default)
rustc 1.8.0 (db2939409 2016-04-11)
```

**Comment by Diggsey** [comment]
Duplicate of #176 and fixed by #421 which will come into effect with the next release.

**Comment by vasily-kirichenko** [comment]
Thanks!
